python/pyspark/mllib/stat/__init__.py

```diff
@@ -21,8 +21,9 @@
 
 from pyspark.mllib.stat._statistics import Statistics, MultivariateStatisticalSummary
 from pyspark.mllib.stat.distribution import MultivariateGaussian
-from pyspark.mllib.stat.test import ChiSqTestResult
+from pyspark.mllib.stat.test import ChiSqTestResult, KolmogorovSmirnovTestResult
 from pyspark.mllib.stat.KernelDensity import KernelDensity
 
-__all__ = ["Statistics", "MultivariateStatisticalSummary", "ChiSqTestResult",
+__all__ = ["Statistics", "MultivariateStatisticalSummary",
+           "ChiSqTestResult", "KolmogorovSmirnovTestResult",
```

Comment:
These modifications where made, to add test documentation in the stats section. Additionally, it is rather inconsistent to export one test result class and not other.